### PR TITLE
DEV-3350: Move log detail to debug level

### PR DESCRIPTION
**Pull request by nrh-cklimas**
Please fill out each section even if it's just with "N/A"

#### Plan? (if this draft/incomplete indicate what you intend to do and how)

n/a

#### What's this PR do?

I did a search in our code for `logger.info()` calls and shifted calls that appeared to be logging dictionaries into `logger.debug()`. I kept the `logger.info()` calls around but shortened what they logged.

#### Why are we doing this? How does it help us?

Makes logs less verbose and helps with Papertrail quotas.

#### Are there detailed, specific, step-by-step testing instructions in the associated ticket?

Yes.

#### Did this PR make changes to the user interface that may require changes to the user-facing docs?

No.

#### Have automated unit tests been added? If not, why?

No, because this affects logging only.

#### How should this change be communicated to end users?

Not needed.

#### Are there any smells or added technical debt to note?

No.

#### Has this been documented? If so, where?

No.

#### What are the relevant tickets? Add a link to any relevant ones.

https://news-revenue-hub.atlassian.net/browse/DEV-3350

#### Do any changes need to be made before deployment to production (adding environment variables, for example)? If so, open a ticket and link it to the ticket for this PR and list it here:

No.

#### Are there next steps? If so, what? Have tickets been opened for them? List next-step tickets here:

No.

### CODE REVIEW

**Review by benjaminEwhite — COMMENTED** [review]
Overall looks good. Will expand on this in JIRA, but should coordinate with Leo re: standardization of logging approach and prepending function names (or not).

**Comment by nrh-cklimas** [comment]
@benjaminEwhite I tweaked the log messages in 1845314b1fd31a87e7b3d1939b5e9d68b4de6007.

**Review by benjaminEwhite — APPROVED** [review]

File: apps/organizations/serializers.py [hunk 1]
```diff
@@ -138,7 +138,8 @@ class Meta:
     def update(self, instance, validated_data):
         """We override `.update` so we can pass update_fields to `instance.save()`. We have code that creates mailchimp entities
         if mailchimp_list_id is being updated. Beyond that, `update_fields` guards against race conditions."""
-        logger.info("Updating RP %s with data %s", instance, validated_data)
+        logger.info("Updating RP %s", instance)
```
Comment (njharman) [inline apps/organizations/serializers.py:141]: verifying "instance" is something like django model instance and will just output str() or repr() and not something large, right?

File: apps/organizations/tasks.py [hunk 2]
```diff
@@ -56,8 +56,9 @@ def exchange_mc_oauth_code_for_mc_access_token(oauth_code: str) -> str:
         "code": oauth_code,
     }
 
-    logger.info(
-        "exchange_mc_oauth_code_for_mc_access_token making a request to Mailchimp with the following data: %s",
+    logger.info("making a request to Mailchimp")
+    logger.debug(
+        "making a request to Mailchimp with the following data: %s",
         request_data | {"code": "REDACTED", "client_secret": "REDACTED"},
```
Comment (njharman) [inline apps/organizations/tasks.py:62]: clever

File: apps/organizations/tasks.py [hunk 3]
```diff
@@ -56,8 +56,9 @@ def exchange_mc_oauth_code_for_mc_access_token(oauth_code: str) -> str:
         "code": oauth_code,
     }
 
-    logger.info(
-        "exchange_mc_oauth_code_for_mc_access_token making a request to Mailchimp with the following data: %s",
+    logger.info("making a request to Mailchimp")
```
Comment (njharman) [inline apps/organizations/tasks.py:59]: I question the value of "making a request" log at .info level...

**Review by njharman — CHANGES_REQUESTED** [review]
Couple minor comments. Request a look, no need for rereview.

Also was this limited to just those apps.
I seem similar logs 

1. "request.data" users/views.py:297
2. "data" common/hookdeck.py:35
3. "request.data" api/views.py:155, 160, 213
4. "template.data" emails/tasks.py:207
5. "payload" organizations/views.py:208
6. "validated_data contributions/serilizsers.py:687
stopped looking here

File: apps/organizations/serializers.py (hunk 1 above)
Comment (nrh-cklimas) [reply apps/organizations/serializers.py:141]: I checked this class (and others) and they have `__str__` methods, so I believe so.

File: apps/organizations/tasks.py (hunk 3 above)
Comment (nrh-cklimas) [reply apps/organizations/tasks.py]: Removed in 239fafd5dc696a5bbd3dd0d55497e00b78a64476

**Review by njharman — APPROVED** [review]
Ship it!

**Pull request merged**